content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

```diff
@@ -1,77 +1,107 @@
 
-Let’s take an example with a product - a table - configured with a
-*FIFO costing method* and an automated inventory valuation.
+Receive a product
+-----------------
+For example, the product, table, is categorized as `Office Furniture` in the product's
+:guilabel:`Product Category` field. In the product category, set the :guilabel:`Costing Method` as
+`First In First Out (FIFO)` and the :guilabel:`Inventory Valuation` field as `Automated`.
 
-I purchase 10 tables at a cost of $10.
+Next, purchase
+10 tables are purchased, each costing $10, as detailed in the :abbr:`PO(Purchase Order)` below.
 
 .. image:: using_inventory_valuation/use_inventory_valuation_01.png
     :align: center
 
-When I’ll confirm the receipt of the products, the value of my inventory
-will be impacted. If I want to know what this impact is, I can click on
-the valuation stat button.
-
-.. tip::
-         The consignment feature allows you to set owners on your stock (discover
-         more about the consignment feature). When you receive products that are
-         owned by another company, they are not taken into account in your
-         inventory valuation.
+After selecting :guilabel:`Validate`, use the :guilabel:`Valuation` smart button to view how the
+value of inventory was impacted.
 
 .. image:: using_inventory_valuation/use_inventory_valuation_02.png
     :align: center
 
-.. note::
-         You need access rights on the accounting module to see that button.
+.. important::
+         :doc:`Developer mode<../../../../general/developer_mode>` must be turned on to see that
+         button.
 
-In this case, I can see that the 10 tables entered the stock for a total
-value of $100.
+.. tip::
+         The :doc:`consignment<../misc/owned_stock>` feature allows you to set owners to items in
+         your stock. Thus, products owned by other companies are not accounted for in your inventory
+         valuation.
+
+The :guilabel:`Stock Valuation` page displays valuation of all products in the shipment-- in this
+case, :guilabel:`Receipt` `WH/IN/00001`. Since there is only 1 type of product, the `Table`, only 1
+record is displayed, indicating that the 10 new tables entering the stock value $100.
 
 .. image:: using_inventory_valuation/use_inventory_valuation_03.png
     :align: center
 
-I can also easily access the accounting entry that has been generated
-(in case of automated inventory valuation).
+In Odoo, automatic inventory valuation records are also recorded in the :guilabel:`Accounting` app.
+To access these accounting entries,navigate to :menuselection:`Accounting --> Journal Entries`, and
+look for `Inventory Valuation` entries with the `STJ` prefix in the :guilabel:`Journal` and
+:guilabel:`Number` columns, respectively.
+
+For this example, search the :guilabel:`Reference` column for the :guilabel:`Receipt` `WH/IN/00001`
+to select the record for the inventory valuation of 10 tables. Below is the accounting entry.
```

Comment:
a couple points to make here:

1. we can remove the :guilabel: & capitalization from _receipt_
2. "Below is the accounting entry." This text relies on the image to provide context, when the text should really provide all of the needed context even without an image. Lets reword this to follow this principle: 
> For this example, search the :guilabel:`Reference` column for the receipt `WH/IN/00001` to select the record for the inventory valuation of 10 tables and view the accounting entry.